src/RequestHeaderParser.php

```diff
@@ -29,7 +29,7 @@ public function feed($data)
             try {
                 $this->parseAndEmitRequest();
             } catch (Exception $exception) {
-                $this->emit('error', [$exception]);
+                $this->emit('error', [$exception, $this]);
```

Comment:
Not a fan of this tbh. This should probably be discussed in the Stream component instead, but this messes with forwarding events.